teleop_twist_keyboard.py

```diff
@@ -1,13 +1,46 @@
 #!/usr/bin/env python
-import roslib; roslib.load_manifest('teleop_twist_keyboard')
-import rospy
 
-from geometry_msgs.msg import Twist
+# Copyright 2011 Brown University Robotics. All rights reserved.
```

Comment:
This has been modified by you so the copyright should be extended with 2017 Open Source Robotics Foundation Inc.